x/concentrated-liquidity/lp_test.go

```diff
@@ -339,6 +367,7 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 		"base case: withdraw full liquidity amount": {
 			setupConfig: baseCase,
 			sutConfigOverwrite: &lpTest{
+				// Subtracting one to account for rounding down in favor of the pool.
 				amount0Expected: baseCase.amount0Expected, // 0.998976 eth
 				amount1Expected: baseCase.amount1Expected, // 5000 usdc
```

Comment:
Where do we subtract one here?

Reply:
I was first going to add one to the shared test case between add and withdraw. However, ended up keeping the base case as is, making this comment obsolete.
Sorry for the confusion